Library/Formula/glibc.rb

```diff
@@ -19,6 +21,8 @@ def install
         "--prefix=#{prefix}",
         "--enable-obsolete-rpc",
         "--without-selinux"] # Fix error: selinux/selinux.h: No such file or directory
+      args << "--enable-kernel=" +
+        `uname -r | cut -d. -f1,2,3 | cut -d- -f1` if build.with? "current-kernel"
```

Comment:
That's one pipe instead of two - that's a plus. It relies on a dash in the kernel version -- that's a minus. From [here](https://www.kernel.org/category/releases.html)

> It is easy to tell if you are running a distribution kernel. Unless you downloaded, compiled and installed your own version of kernel from kernel.org, you are running a distribution kernel. To find out the version of your kernel, run uname -r:
> 
> $ uname -r
> 3.7.5-201.fc18.x86_64
> 
> **If you see anything at all after the dash**, you are running a distribution kernel. Please use the support channels offered by your distribution vendor to obtain kernel support.
> 

So, relying on a dash in the return value of `uname -r` we put aside those two and a half enthusiasts who compile Linux kernel themselves. But I'm fine with this version if you insist because if someone compiles a kernel - he or she can figure out how to fix this formula as well.